piksi_tools/console/settings.yaml

```diff
@@ -228,10 +228,10 @@
 - group: solution
   name: soln_freq
   type: integer
-  units: hz
+  units: Hz
   default value: '10'
   enumerated possible values:
-  Description: The rate at which a solution is generated internally to the Piksi.
+  Description: The frequency rate at which a location solution is computed.
```

Comment:
"frequency rate" is redundant. Please change to "frequency"

Also, do you mean to say "location" solution as opposed to position solution?